.github/workflows/internal-images.yml

```diff
@@ -61,28 +55,30 @@ jobs:
           password: ${{ secrets.GHCR_TOKEN }}
 
       # We need QEMU and Buildx for multi-platform (amd64+arm64) image push. See RATIONALE.md
-      - name: Setup QEMU
+      - name: "Setup QEMU"
         uses: docker/setup-qemu-action@v1
 
-      - name: Setup Buildx
+      - name: "Setup Buildx"
         uses: docker/setup-buildx-action@v1
 
-      # This resolves GOROOT_NAME (ex GOROOT_1_17_X64) into GO_MINOR_REVISION (ex 1.17.1), by
-      # parsing the directory path GOROOT_NAME evaluates to.
-      #
-      # Ex. Given ${!GOROOT_NAME}=/opt/hostedtoolcache/go/1.17.1/x64 -> GO_MINOR_REVISION=1.17.1
-      - name: Export Go minor revision
-        run: |
-          rev=$(echo ${!GOROOT_NAME}|cut -d/ -f5)
+      - name: "Checkout"
+        uses: actions/checkout@v2
+
+      - name: "Configure Go"
+        run: .github/workflows/configure_go.sh
+
+      - name: "Export Go minor revision"
+        run: |  # Ex. GOROOT=/opt/hostedtoolcache/go/1.17.1/x64 -> GO_MINOR_REVISION=1.17.1
```

Comment:
so this is the part that syncs the docker build with the runner's version of go. In a later PR we would refactor this to address the "upgrade go in one PR" problem. However, doing that now would make this change too complex.